data/korath/korath ships.txt

```diff
@@ -270,6 +270,76 @@ ship "Korath Raider Mark XLII"
 	description `Constant warfare against surrounding aliens has been building pressure on Korath shipyards to recover some of their ancient prowess. Now, with a fresh industrial network to support their endeavors, the shipwrights of the Korath have updated the design to deal with modern targets.`
 
 
+ship "Korath Ravager"
+	sprite "ship/ravager"
+	thumbnail "thumbnail/ravager"
+	attributes
+		category "Heavy Warship"
+		"cost" 3760000
+		"shields" 25000
+		"hull" 6500
+		"required crew" 277
+		"bunks" 319
+		"mass" 1260
+		"drag" 12.3
+		"heat dissipation" .7
```

Comment:
This one I'm still iffy about, since it's a very high heat dissipation for a Korath ship. It seems to run just fine at .7 but I'll give everyone some time to throw around suggestions